contrib/jdbc_fdw/Trunk.toml

```
@@ -0,0 +1,23 @@
+[extension]
+name = "jdbc_fdw"
+
+
+version = "0.0.1"
```

```suggestion
version = "1.2.0"
```
Let's update this to match the extension version